### DISCUSSION TITLE
Damage encrypted APFS recovery ( superblock or keybag might be overwritten?

### DISCUSSION THREAD

**Discussion by gaspard-art** [jivanpal/apfs-tools#86] [General]
My external drive(Transcend 2TB HDD) with 1.5TB APFS encrypted volume, there are many photo library files in it. One day I used 0.5TB fat32 part as windows 11 install disk, after I finish setup, I found that HDD drive only have 32GB, and 1.5TB APFS encrypted volume has gone!!!! I use r-studio and UFS explorer in windows and also try Drat 0.1.3 to scan to find back superblock and keybag, but have no luck. How can I get back pics and photo library in APFS????I think it might be the same issue because I try to find recovery company but expensive and try hex to see metadata inside it seem garbied code???I know my original password~~
![螢幕擷取畫面 2024-10-01 001645](https://github.com/user-attachments/assets/fff97bd3-4cdc-4779-ba35-d839f8505448)
![螢幕擷取畫面 2024-10-01 002240](https://github.com/user-attachments/assets/a0f0cdc0-511d-40ec-a3d3-451bcc4f4735)
![螢幕擷取畫面 2024-10-01 002254](https://github.com/user-attachments/assets/78b8519f-d7b9-43a2-8cf8-c1804c30a82c)
![螢幕擷取畫面 2024-10-01 002315](https://github.com/user-attachments/assets/f3a11af1-8bc2-4b87-a3bd-fcba8cd9be16)
![螢幕擷取畫面 2024-10-01 002334](https://github.com/user-attachments/assets/b7a18370-a29b-405a-ac55-580bcd4e92e5)
![螢幕擷取畫面 2024-10-01 002556](https://github.com/user-attachments/assets/e226f2eb-6f8f-4192-809e-939d45da7a2a)

**Comment by jivanpal** [comment]
Sorry, but Drat does not support encrypted APFS at this time.
